utils.jl

```diff
@@ -7,15 +7,14 @@ macro gctime(ex)
         n = isempty(ARGS) ? 1 : parse(Int, ARGS[1])
         local times = Vector{Float64}(undef, n)
         local vals = Vector{Any}(undef, n)
-        local stats = Base.gc_num()
+        local stats = Vector{Any}(undef, n)
         t0 = time_ns()
         for i in 1:n
             vals[i] = $(esc(ex))
             times[i] = time_ns()
+            stats[i] = Base.gc_num()
         end
-        local diff = Base.GC_Diff(Base.gc_num(), stats)
-        @. times = (times - t0) / 1e9
-        result = (value=vals, times=times, bytes=diff.allocd, gctime=diff.total_time/(1e9*n), gcstats=diff)
+        result = (value=vals, times=times, stasts=stats)
```

Comment:
this should probably be combined with https://github.com/JuliaCI/GCBenchmarks/pull/7 to make the times diff properly.